component/tests/defaults.yml

```diff
@@ -29,3 +29,50 @@ parameters:
               schema:
                 openAPIV3Schema:
                   type: object
+
+    compositions:
+      test-composition:
```

Comment:
Since we plan to have a lot of composistions here, does it make sense to group them by domain, like `s3-compositions`?

Reply:
I'm not sure what that grouping by domain would actually achieve, Where do you see the value?
I mean we could just prefix the name of compositions according to their domain.

Reply:
It will be easier to figure out where the compositions belong to.

Reply:
Please have a look at the next PR that adds the cloudscale composition: https://github.com/vshn/component-appcat/blob/pkg-composition/packages/composition/objectstorage/cloudscale.yml#L8-L13
Is it not enough to figure where it belongs? It's in the spec which XRD is being implemented.

Reply:
I will go forward with the current implementation for now. We can still discuss this topic in the next stage.